src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -79,7 +79,8 @@ class PartitionSection < ::Installation::AutoinstProfile::SectionWithAttributes
         { name: :bcache_backing_for },
         { name: :bcache_caching_for },
         { name: :device },
-        { name: :btrfs_name }
+        { name: :btrfs_name },
+        { name: :quotas }
```

Comment:
I was thinking about it but `quotas` feels more natural to me. However, it is true that `btrfs` uses `quota`, so I guess I should change it to `quota`.

Reply:
Finally, I have decided to keep the `quotas` name but only in the `DriveSection`. Internally, we will use `quota` (which is not a good name in my opinion), but the user will read `quotas` (like in *Subvolumes Quotas*). I am not using `subvolmes_quotas` because, who knows, maybe in the future we add support for quotas in other file systems.